### ISSUE TITLE
Checklist Model

### ISSUE THREAD

**Issue by Pherkel** [Algo-Boys/SWR2-ASR#36]
- [x] Meaningful variable names
- [x] PEP8 compliant (Pylint value of 9.0 or higher)
- [x] One line no more than one logical step
- [x] Where do you get the data from?
- [x] Deterministic train, validation, test split
- [x] Where to put the data?
- [x] Is the code reproducible (in a loose sense)? Can I run it without extra knowledge?
- [x] Docstrings that describe what is/should be achieved and how to use? (Not how it is implemented.)
- [x] Minimal comments; the code should be self-explanatory
- [x] Split into train validation test set
- [x] Training loop (**2 points**)
- [x] Storing weights and continuing from checkpoint
- [x] Code for plotting of loss reduction over epochs
- [x] Code for evaluation measure (figure / table)
